new Wasteplanet Ruin [The Ship Graveyard]

spockye opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request


This PR adds a new wasteplanet ruin, The Ship Graveyard!
--


---

it's an old Scrapyard, which was abandoned. A lot of the Old yardcrew were left behind, and are now fighting within the old hulking wrecks. Will you be the one to take control over it, and extract the old treasures?

![2023-09-29 20 03 06](https://github.com/shiptest-ss13/Shiptest/assets/79304582/4083bbfe-6410-4b3b-800f-c0ab513759eb)


![2023-09-29 20 04 25](https://github.com/shiptest-ss13/Shiptest/assets/79304582/93632cab-882b-497e-97d1-45ee8921142e)


---

LOOT:
1x AKSU
1x M90
2x polymer rifle
1x winchester rifle
1x sabre
1x sci imprinter
1x commander pistol
a lot of misc scrap
misc minor loot
a Tattered Centcom Formal Coat

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

we NEED new wasteplanet ruins
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Added a new wasteplanet ruin (ship_graveyard)
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

spockye committed:
Create wasteplanet_ship_graveyard.dmm

spockye committed:
ye

wo

spockye committed:
woop, fixes and changes!

spockye committed:
zone and map fixes

spockye committed:
misc loot changes and stuff

im pretty tired. should be fine
lets run checks on this bad boy

spockye committed:
Merge branch 'master' into the-shipgraveyard

spockye committed:
fixes button and shutter names

spockye committed:
Merge branch 'the-shipgraveyard' of https://github.com/spockye/Shiptest into the-shipgraveyard

spockye committed:
fixes last name errors

spockye commented:
don't you dare fail

meemofcourse commented:
*fails ur checks

spockye committed:
Merge remote-tracking branch 'upstream/master' into the-shipgraveyard

spockye committed:
Update wasteplanet_ship_graveyard.dmm

spockye committed:
Merge remote-tracking branch 'upstream/master' into the-shipgraveyard

spockye committed:
Merge branch 'master' into the-shipgraveyard

goober3 reviewed (changes requested):
![2107328b7d2dc7d6c5d543950e5cf05b](https://github.com/shiptest-ss13/Shiptest/assets/118859017/89edf47e-9781-4c0a-898d-bf47753cc5f2)
![607a84ecdf4180b29892e87e9fa4334b](https://github.com/shiptest-ss13/Shiptest/assets/118859017/b00e8204-388c-4b19-a25f-519cd986e5bc)
Unrotated Airlocks


![image](https://github.com/shiptest-ss13/Shiptest/assets/118859017/65804c4b-2443-442a-83f4-c76c7418ff36)
![d4123ffdee219c1f6a44134932ec10d9](https://github.com/shiptest-ss13/Shiptest/assets/118859017/71cc29f6-4cf0-49e4-8b19-d244cdefe759)
![7fa6d614b7c5d1e636da5bcccfc8e448](https://github.com/shiptest-ss13/Shiptest/assets/118859017/71fb48b3-e216-4051-99d3-effabeb24e72)
![4542687d01fdd6f0dccb216bd0f2002e](https://github.com/shiptest-ss13/Shiptest/assets/118859017/54549ac1-1d9b-4189-ba20-7551cfacc7b1)
Stacked Wallmounts


![b4528a373fbb40feed293ec58dd7a152](https://github.com/shiptest-ss13/Shiptest/assets/118859017/be9dd759-28fd-4954-ac87-34349cd6456e)
Floating Extinguisher Cabinet


![7ad536e94ef489945a8c7695d82d1c2e](https://github.com/shiptest-ss13/Shiptest/assets/118859017/183377eb-acd8-4661-9b42-fb37a9dc9ae3)
Floating Light


![c3c74eae8f25acfb3ed42f27b8d71442](https://github.com/shiptest-ss13/Shiptest/assets/118859017/7202f7a6-fc9d-433b-8a63-9655cd5004c1)
Stray Pipes. it would be so real if you fully atmos'd these ships.

![6be3fcf9190d3fedf6816d0b8f1bb15a](https://github.com/shiptest-ss13/Shiptest/assets/118859017/0b426108-fff5-4a05-8d29-6f00afc3c02d)
add corners to these warning stripes, please!!

![8ccf249e7b776af33ce6c16988950317](https://github.com/shiptest-ss13/Shiptest/assets/118859017/eba37eea-3482-4182-93ed-2c01ed5bfa54)
This Seems Excessive! 

![4753a800447b7a8aaf6949f1a33e62af](https://github.com/shiptest-ss13/Shiptest/assets/118859017/d6b06c67-f659-487e-85a9-86f47072a416)
tone down the loot in this room, please! one pair of magboots should do fine.

![35f670b03ad610307d97fd05b93085e0](https://github.com/shiptest-ss13/Shiptest/assets/118859017/fc8bd1b0-d1c1-455b-b57d-f6a0838281a3)
the elusive Wall Mattress.

goober3 commented on code/game/area/areas/ruins/wasteplanet.dm:
```diff
@@ -43,3 +43,19 @@
+	icon_state = "blue"
+
+/area/ruin/wasteplanet/wasteplanet_graveyard/syndicate_ship //ruined cargo hauler, dont mind this
+	name = "Scrapped Syndicate Haluer"
```
haluer

spockye replied:
:c

Latentish commented on _maps/map_catalogue.txt:
```diff
@@ -609,3 +609,7 @@ Find the key for using this catalogue in "map_catalogue_key.txt"
+
+		File name = "_maps\RandomRuins\wasteruins\wasteplanet_ship_graveyard"
+		Size = (x = 102)(y = 75)(z = 1)
+		Tags = "Medium Loot", "Shelter", "Hazardous"
```
You need to add a combat challenge tag.

spockye committed:
Merge remote-tracking branch 'upstream/master' into the-shipgraveyard

spockye committed:
responds to reviews

fixes the txt
makes the misc map fixes
Hauler..... :)

also adds some more broken lights

spockye committed:
boss, not HIGH

goober3 reviewed (changes requested):
![7f45db3d39ca3eea8d0ffe1655c8513b](https://github.com/shiptest-ss13/Shiptest/assets/118859017/8450e2b9-38e2-47e2-8634-0c976b208f50)
![3bcc7aef70de6801ccdc89138c83843b](https://github.com/shiptest-ss13/Shiptest/assets/118859017/ba809792-559b-471d-b3ef-6b526ecfd3aa)
Oops ! More Extinguishers. (patch up the turf passthrough, also)

spockye committed:
goob review

Erikafox reviewed (changes requested):
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/a080a5e7-9665-4c5b-b54a-951727630348)
just use one iron ore and varedit the amount var

![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/e20bf626-842f-4942-b78a-8e66eeee95ba)
Add some scattered cable coils / scrap, show it was Being Disassembled

![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/d3b6d56b-1f5c-4c7c-a31a-62c7c473d271)
Why is this here (outside the scrap yard), why is it completely intact. Why is it packed with loot?
Bad wallmount on the newscaster.
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/60b2d983-377a-4461-914f-3560d5fce4da)
Use some frontiersmen tbh
![image](https://github.com/shiptest-ss13/Shiptest/assets/94164348/a9c098dd-c840-4e67-811c-5da49eef194d)
Good god no. 

Overall needs another pass to make it scrappier + consider introducing my favorite things: Hivebots, they literally scavenge and break stuff down so they'd fit well here.

spockye committed:
responds to review

removes the shippy and its area

adds some more junk around

adds some more hivebots

fixy fixy fixy
